plugins/backup/veeam/src/main/java/org/apache/cloudstack/backup/VeeamBackupProvider.java

```diff
@@ -174,7 +174,7 @@ public boolean removeVMFromBackupOffering(final VirtualMachine vm) {
             if (!client.removeVMFromVeeamJob(vm.getBackupExternalId(), vm.getInstanceName(), vmwareDC.getVcenterHost())) {
                 LOG.warn("Failed to remove VM from Veeam Job id: " + vm.getBackupExternalId());
             }
-        } catch (CloudRuntimeException e) {
+        } catch (Exception e) {
             LOG.debug("VM was removed from the job so could not remove again, trying to delete the veeam job now.", e);
```

Comment:
can't we catch a veeam specific exception? anything may pass as this now (mem/npe/io/...)